content/en/docs/tasks/administer-cluster/quota-api-object.md

```diff
@@ -132,6 +132,14 @@ by quotas:
 <tr><td>"services.loadbalancers"</td><td>Service of type LoadBalancer</td></tr>
 </table>
 
+The 1.9 release added support for quotas on built-in kubernetes objects.
+Additionally, the 1.15 release added support for custom resources using the same syntax:
+
+* `count/<resource>.<group>`
```

Comment:
Can you provide a full example of how this is used? I'm not sure I understand by reading this snippet. Thanks!

Reply:
Maybe this extra text makes sense in a What's Next section?